var/spack/repos/builtin/packages/intel-mkl/package.py

```diff
@@ -72,3 +74,6 @@ class IntelMkl(IntelPackage):
     if sys.platform == 'darwin':
         # there is no libmkl_gnu_thread on macOS
         conflicts('threads=openmp', when='%gcc')
+
+    def setup_build_environment(self, env):
+        env.prepend_path('PATH', self.spec['cpio'].prefix.bin)
```

Comment:
This shouldn’t be necessary, build deps are automatically added to the PATH

Reply:
Fixed, thanks.